config/default/kustomization.yaml

```diff
@@ -36,6 +37,18 @@ patches:
 # through a ComponentConfig type
 #- manager_config_patch.yaml
 
+# [PEERPODS] To enable the peerpod controllers uncomment this section.
```

Comment:
would it make sense to have `config/default-with-peerpods` overlay that adds all the peerpods configs on top of `default`. It'd prefer to have the rbac settings maintained part of that overlay too.